capsules/src/st77xx.rs

```diff
@@ -238,15 +238,17 @@ impl<'a, A: Alarm<'a>, B: Bus<'a>, P: Pin> ST77XX<'a, A, B, P> {
         bus: &'a B,
         alarm: &'a A,
         dc: Option<&'a P>,
-        reset: &'a P,
+        reset: Option<&'a P>,
         buffer: &'static mut [u8],
         sequence_buffer: &'static mut [SendCommand],
         screen: &'static ST77XXScreen,
     ) -> ST77XX<'a, A, B, P> {
         if let Some(dc) = dc {
             dc.make_output();
         }
-        reset.make_output();
+        if let Some(reset) = reset {
+            reset.make_output();
+        }
```

Comment:
```suggestion
        reset.map(|reset| reset.make_output());
```
I consider this slightly cleaner, but its up to you

Reply:
Good point, I updated `dc` and `reset` to map.